audio-analysis/lib/etl/sqlConnection.py

```diff
@@ -15,7 +15,7 @@ def getEngine(isTest):
             engine.connect()
             print("\nYou are connected to ", engine, "\n")
         except Exception:
-            engine = create_engine('mysql+pymysql://root@localhost/climactic_test')
+            engine = create_engine('mysql+pymysql://root:root@localhost:8889/climactic_test')
             engine.connect()
             print("\nYou are connected to ", engine, "\n")
         except:
```

Comment:
I see you removed the `root@localhost` engine creation in _sqlConnection.py_
This causes a break from my end. When you do this it workks for me:
```
         try:
            engine = create_engine('mysql+pymysql://root@localhost/climactic_test')
            engine.connect()
            print("\nYou are connected to ", engine, "\n")
        except Exception:
            engine = create_engine('mysql+pymysql://root:root@localhost/climactic_test')
            engine.connect()
            print("\nYou are connected to ", engine, "\n")
        except Exception:
            engine = create_engine('mysql+pymysql://root:root@localhost:8889/climactic_test')
            engine.connect()
            print("\nYou are connected to ", engine, "\n")
        except:
            print("\n ERROR: Issues connecting to your localhost \n")
 ```

Reply:
oop my bad good sir. Fixed.

Reply:
Nice, looks good now